### PR TITLE
feature<BookList>: add book list template

### PR DESCRIPTION
**Pull request by sergejvoronov**

### CODE REVIEW

File: src/MasterPeace/Bundle/BookBundle/Resources/views/Book/list.html.twig [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
```
Comment (povils) [inline src/MasterPeace/Bundle/BookBundle/Resources/views/Book/list.html.twig]: is esmes turesite base.html,twig, kuri sis template extendins :)

**Comment by povils** [comment]
Testas?

File: src/MasterPeace/Bundle/BookBundle/Resources/views/Book/list.html.twig [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
```
Comment (sergejvoronov) [inline src/MasterPeace/Bundle/BookBundle/Resources/views/Book/list.html.twig]: Bootstrap template :)

**Comment by sergejvoronov** [comment]
testas kitam branche, kuri dar ne upstreaminau, nes procese ;)

**Comment by KarolisM** [comment]
+1

**Comment by povils** [comment]
:+1:

**Pull request merged**